Signal/src/view controllers/FullImageViewController.m

```diff
@@ -128,56 +159,106 @@ - (void)populateImageView:(UIImage *)image {
 }
 
 - (void)initializeGestureRecognizers {
-    self.doubleTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(imageDoubleTapped:)];
-    self.doubleTap.numberOfTapsRequired = 2;
-
-    self.singleTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(imageSingleTapped:)];
-    [self.singleTap requireGestureRecognizerToFail:self.doubleTap];
-
-    self.singleTap.delegate = self;
-    self.doubleTap.delegate = self;
+    UITapGestureRecognizer *singleTap = [[UITapGestureRecognizer alloc] initWithTarget:self
+                                                                                action:@selector(imageDismissGesture:)];
+    singleTap.delegate = self;
+    [self.view addGestureRecognizer:singleTap];
+    
+    UITapGestureRecognizer *doubleTap = [[UITapGestureRecognizer alloc] initWithTarget:self
```

Comment:
Here's my argument: 

* 90% of the time, users who double-tap want to dismiss.
* People rarely want to zoom photos.  
* If People do want to zoom into photos, they can pinch.  Pinching is a far better interaction for zoom - it's continuous, has continuous feedback, is reversible, let's users pan and scale at the same time, etc.  Double-tap just zooms into a predetermined scale.  Users will almost certainly have to pinch afterward to scale/pan to the portion of the image of interest.
* There's a huge fraction of users who never learn to distinguish single-tap and double-tap.  They double tap everything.  

Is that persuasive?